eth.c

```diff
@@ -0,0 +1,232 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+// SPDX-License-Identifier: GPL-2.0-only
+
+#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
```

Comment:
nit: with dev_*() print functions, modname probably isn't useful since the device name will be in the prefix